### PR TITLE
added noisy relu activation function

### PR DESCRIPTION
**Pull request by arp95**
I added the noisy relu activation function as described over here:
https://en.wikipedia.org/wiki/Rectifier_(neural_networks)

Please review. @rthadur

### CODE REVIEW

**Comment by arp95** [comment]
@tanzhenyu please review.

File: tensorflow/python/keras/layers/advanced_activations.py [hunk 1]
```diff
@@ -330,3 +330,41 @@ def get_config(self):
...
   def compute_output_shape(self, input_shape):
     return input_shape
+
+
+@keras_export('keras.layers.NoisyReLU')
+class NoisyReLU(Layer):
+  """Noisy Rectified Linear Unit activation function.
+
+  It follows:
+  `f(x) = x+Y for x > 0, where Y is noise`,
```
Comment (tanzhenyu) [inline tensorflow/python/keras/layers/advanced_activations.py]: Isn't Y a Gaussian noise? why is it a constant here?

Comment (arp95) [reply tensorflow/python/keras/layers/advanced_activations.py]: I just updated the comment, but I am using noise from the user end.

Comment (tanzhenyu) [reply tensorflow/python/keras/layers/advanced_activations.py]: I mean isn't Y a Gaussian noise such that every time ._call_() happens it should be a different value?

Comment (arp95) [reply tensorflow/python/keras/layers/advanced_activations.py]: The noise value is between 0 to standard deviation of the inputs given. It is upon the user how much noise is to be added to the actual relu function.

**Comment by tanzhenyu** [comment]
Please add to addon. This is our new place to consolidate APIs before moving to tf core.

**Comment by tanzhenyu** [comment]
Let me know if you would like to have code review for add-ons.

**Comment by rthadur** [comment]
I'm going to go ahead and close this PR, because it seems to have stalled. If you're still interested in pursing this (and responding to my comments), please feel free to reopen!

**Pull request closed without merging**